controls/radgridview/export/export.md

```diff
@@ -106,11 +100,8 @@ The following example shows how to show a save file dialog asking the user to sa
 	 End If
 	End Sub
 {{endregion}}
-
->When opening an exported ".xls" file with Excel 2007 or later, you will get a warning message. The reason for it is that the RadGridView can export to HTML and ExcelML formats - in both cases not native for Excel 2007. In case of exporting to HTML format you will get such warning, however if you export to ExcelML format with XML extension you will not get it.
         
-
-In addition,__RadGridView__ provides a built-in methods to get the content of your grid view control in different formats:
+In addition,__RadGridView__ provides built-in methods to get the content of your GridView control in different formats:

```

Comment:
Missing interval prior RadGridView. Consider adding links to each article in the below list.